symengine/eval_mpc.cpp

```diff
@@ -232,15 +232,23 @@ class EvalMPCVisitor : public BaseVisitor<EvalMPCVisitor> {
         } else if (x.__eq__(*E)) {
             mpfr_t t;
             mpfr_init2(t, mpc_get_prec(result_));
-            mpfr_const_euler(t, rnd_);
+            mpfr_t one_;
+            mpfr_init(one_);
+            mpfr_set_si(one_,1,rnd_);
+            mpfr_exp(t,one_, rnd_);
             mpc_set_fr(result_, t, rnd_);
             mpfr_clear(t);
+            mpfr_clear(one_);
         } else if (x.__eq__(*EulerGamma)) {
             mpfr_t t;
             mpfr_init2(t, mpc_get_prec(result_));
-            mpfr_const_euler(t, rnd_);
+            mpfr_t one_;
+            mpfr_init(one_);
+            mpfr_set_si(one_,1,rnd_);
+            mpfr_exp(t,one_, rnd_);
             mpc_set_fr(result_, t, rnd_);
             mpfr_clear(t);
+            mpfr_clear(one_);
```

Comment:
Instead of having `t` and `one_` here, use one variable `t` as it is enough in this case.